### PR TITLE
Quantum 301 preview

### PR DESCRIPTION
**Pull request by frankharkins**
## Changes

This PR is to create a preview of [`jhwatrous:quantum-301`](https://github.com/jhwatrous/platypus/tree/quantum-301) to share with others. Unfortunately, #940 is skipping the preview, so I've pulled in the changes here.

[Link](https://platypus-pr-942.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/basics/)

### CODE REVIEW

**Comment by pandasa123** [comment]
For the following line:

> Suppose that we have a system — meaning an abstraction of a physical device or a medium of some sort — that stores information

I feel the definition of `system` could be added in the glossary instead? Simplifying this as:

> Suppose that we have a system that stores information

And allowing readers who don't immediately recognise the concept of a system to hover over for context

File: notebooks/toc.yaml [hunk 1]
```diff
@@ -1,3 +1,19 @@
+- title: Basics of quantum information
+  url: /basics
+  sections:
+    - title: Single systems
+      id: single-systems
+      url: /basics/single-systems
+
```
Comment (vabarbosa) [inline notebooks/toc.yaml:7]:
```suggestion
```

File: notebooks/toc.yaml [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+    - title: Multiple systems
+      id: multiple-systems
+      url: /basics/multiple-systems
+
```
Comment (vabarbosa) [inline notebooks/toc.yaml:11]:
```suggestion
```

**Comment by techtolentino** [comment]
@frankharkins - do we still need this branch, especially since we already have https://learn.qiskit.org/course/basics/single-systems

**Comment by frankharkins** [comment]
Yes please, we're using it to work on the other upcoming pages

**Pull request closed without merging**